src/Murder/Murder.csproj

```diff
@@ -3,21 +3,46 @@
   <PropertyGroup>
     <TargetFramework>net8.0</TargetFramework>
 
-    <!-- Used for Generator -->
-    <CopyLocalLockFileAssemblies>true</CopyLocalLockFileAssemblies>
-    
+    <AssemblyName>Murder</AssemblyName>
+    <AssemblyVersion>1.0.0.0</AssemblyVersion>
+
+    <Nullable>enable</Nullable>
```

Comment:
It doesn't pick up the properties in the Directory.Build.Props?

Reply:
It should, good catch

Reply:
I think that's also the same for LangVersion, AllowUnsafeBlocks and ImplicitUsings 😁

Reply:
Done!